pxr/base/arch/attributes.h

```diff
@@ -181,6 +181,25 @@ PXR_NAMESPACE_OPEN_SCOPE
 #   define ARCH_USED_FUNCTION __attribute__((used))
 #   define ARCH_EMPTY_BASES
 
+#if defined(ARCH_SANITIZE_ADDRESS)
```

Comment:
Is there harm in just always setting these, not just when the sanitizer is enabled? It would reduce variability.